src/components/EmptyMailbox.vue

```diff
@@ -1,7 +1,6 @@
 <template>
 	<div id="emptycontent">
-		<div class="icon-mail"></div>
-		<h2>{{ t('mail', 'No messages in this folder') }}</h2>
+		<p> <slot /></p>
```

Comment:
Imo this is a bit strange. The `<EmptyMailbox>` is only used once, so the slot value can just be moved here.